libs/barista-components/table/src/table.ts

```diff
@@ -583,6 +585,7 @@ export class DtTable<T> extends _DtTableBase<T> implements OnDestroy {
   ): void {
     switch (typeof val) {
       case 'string':
+        val = val.replace(/[\n\r\t]/g, ' ');
```

Comment:
Please add a comment what you are replacing here with this regex. I am really bad at regex :D